src/browser/app/App.react.js

```diff
@@ -37,7 +43,12 @@ export default class App extends Component {
           titleTemplate="%s - Este.js"
         />
         {/* Pathname enforces rerender so activeClassName is updated. */}
-        <Header msg={msg} pathname={pathname} viewer={viewer} />
+        <Header
+          isLoggedIn={!!authToken}
```

Comment:
Which logic? We have to pass updated pathname to enforece rerender on pure component.